processor/remotetapprocessor/processor.go

```diff
@@ -89,31 +92,40 @@ func (w *wsprocessor) Shutdown(ctx context.Context) error {
 }
 
 func (w *wsprocessor) ConsumeMetrics(_ context.Context, md pmetric.Metrics) (pmetric.Metrics, error) {
-	b, err := metricMarshaler.MarshalMetrics(md)
-	if err != nil {
-		w.telemetrySettings.Logger.Debug("Error serializing to JSON", zap.Error(err))
-	} else {
-		w.cs.writeBytes(b)
+	if w.limiter.Allow() {
```

Comment:
Is there a way to add a test to ensure that the limit is being applied properly?

Reply:
Have already solved.